Add blog on Chrys Woods's talk.

r-j-arnold opened the pull request.

rgaiacs commented on site/posts/hacktoberfest.md:
```diff
@@ -0,0 +1,117 @@
+<br>
+
+
+We'll be honest here and say that our **Sheffield R Users group Hacktoberfest celebrations** started as a last minute stroke of inspiration. Nearing our standard first Tuesday of the month meetup, our speaker lineup for October was thin. At the same time I'd spent the last month mentoring as part of [**Mozilla Open Leadership program**](https://mozilla.github.io/leadership-training/) again, which ...
```
Add link to Sheffield R Users group website.

rgaiacs replied:
I also recommend to not use `[**X**](url)`.

rgaiacs commented on site/posts/hacktoberfest.md:
```diff
@@ -0,0 +1,117 @@
+
+## The sessions
+
+Admittedly, when you work at a computer all day, spending another 3 hours at your screen voluntarily is probably not everyone's top choice. But I personally found the opportunity to carve some time out to explore the huge variety of projects and diverse ways in which to get involved engaging and in some ways quite relaxing. The great collaborative spaces available for booking at the University of ...
```
Use "Hacktoberfest" with capital "H" since we are using this form in other part of the blog post.

rgaiacs commented on site/posts/hacktoberfest.md:
```diff
@@ -0,0 +1,117 @@
+
+<br>
+
+My t-shirt in the end was mainly earned by helping with typos. For Hackoberfest, *the size of your contribution doesn't matter* as long as you send a contribution. And finding typos is actually non-trivial and time consuming due to our brain's auto-correct feature. Sadly, the coding pieces that I worked on over the session did not end up making the cut to submit as a functional PR yet (there'll b ...
```
Replace "PR" with "pull request" here. Otherwise, make clear that PR means pull request before.

rgaiacs commented:
Great blog post. :-) We can republish it at https://www.software.ac.uk/ later if you want. In that case, I will copy e4a9476 for prepare it.

annakrystalli commented:
Thanks @rgaiacs. Incorporated changes. And yeah republishing sounds good!

mikecroucher commented:
Great work!  I'll publish this on Monday.

mikecroucher commented:
Or wednesday (Insert looking sheepish emoticon here)

r-j-arnold committed:
Add blod on Chrys Woods's talk.

r-j-arnold committed:
Remove accidental extra file

r-j-arnold committed:
Remove accidental extra file

willfurnass commented on _posts/2019-01-30-Chris-woods-talk.md:
```diff
@@ -0,0 +1,34 @@
+
+Chris first discussed the question of what does good code for research need to be, and gave three main criteria. 
+
+It needs to be flexible enough to adapt to evolving research questions
```
Make into a bulletted list

r-j-arnold replied:
Done. I

willfurnass commented on _posts/2019-01-30-Chris-woods-talk.md:
```diff
@@ -0,0 +1,34 @@
+
+Chris also talked at length about the importance of testing. This included a section of his “scare the principal investigator talk” covering cases where nonexistent or inadequate testing had huge reputational and financial costs, such as a case where a single incorrect character in a program led to five retractions from major journals including Science. Every program contains bugs, and tests may  ...
+
+He went on to discuss different kinds of test, such as unit tests to check small pieces of code are working as expected, and systems tests which check that the pieces are working together correctly. Chris particularly advocated for runtime tests which test key aspects of the program as it runs. This allows problems to be caught early on, effectively “Minimising the blast radius”. Without runtime  ...
```
"at least one test when they get back to the office"?

r-j-arnold replied:
Done

willfurnass reviewed (changes requested):
Two suggestions but, apart from those, LGTM.  Thanks for the post!

r-j-arnold committed:
Make list bulleted

Formatting error had deleted bullet points, restore them

r-j-arnold committed:
End sentence more neatly 

"write at least one test" -> "write at least one test when they got back to their office"

willfurnass commented:
@r-j-arnold The site can't build unless you enter a valid date in the YAML front matter of the Markdown doc.

r-j-arnold committed:
Add date to YAML info

r-j-arnold commented:
> @r-j-arnold The site can't build unless you enter a valid date in the YAML front matter of the Markdown doc.

@willfurnass Yeah, I just noticed that, hopefully it'll pass now

r-j-arnold commented:
@willfurnass Passed travis, and requested changes have been made
